pyxem/signals/correlation2d.py

```diff
@@ -23,14 +23,17 @@
 from hyperspy.signals import Signal2D
 from hyperspy._signals.lazy import LazySignal
 
-from pyxem.utils.correlation_utils import (
-    corr_to_power,
+from pyxem.utils._correlations import (
     _get_interpolation_matrix,
     _symmetry_stem,
 )
 from pyxem.signals.common_diffraction import CommonDiffraction
 
 
+def corr_to_power(z):
```

Comment:
I would just make this private and then keep it under ` _correlations`

Reply:
can do. will then merge.